data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/sink/LambdaSinkConfig.java

```diff
@@ -18,10 +18,10 @@
 
 public class LambdaSinkConfig {
 
-    public static final String REQUEST_RESPONSE = "RequestResponse";
-    public static final String EVENT = "Event";
-    public static final String BATCH_EVENT = "batch_event";
-    public static final String SINGLE_EVENT = "single_event";
+    public static final String REQUEST_RESPONSE = "request-response";
```

Comment:
Perhaps make a `CommonLambdaConfig` class that has these constants. We should avoid duplicating these or we may have future mismatches.